packages/link-lists/src/sass/module.scss

```diff
@@ -0,0 +1,43 @@
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// link-lists module
+// A simple list of inline links.
+//
+// Content:
+// - .inline-links, .inline-links--inverted
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+
+@import 'globals.scss';
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Link-list styles
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+/**
+ * Block of inline links.
+ */
+.inline-links {
+	display: inline-block;
+	list-style-type: none;
+	margin: 0;
+	padding: uikit-space(0.5) 0;
+
+	> li {
```

Comment:
We do this classless here because a link-list can only be a list. anything else is getting a bit too mad :)